app/src/components/grids/cardsGrid.tsx

```diff
@@ -0,0 +1,55 @@
+"use client";
+
+import React from "react";
+import useBreakpoints from "../../hooks/useBreakpoints";
+import CollectionDataType from "../../types/CollectionDataType";
+import { CollectionCardModel } from "../../models/collectionCard";
+import ItemDataType from "@/src/types/ItemDataType";
+import ItemCardDataType from "@/src/types/ItemCardDataType";
+import { ItemCardModel } from "@/src/models/itemCard";
+import { SimpleGrid as DCSimpleGrid } from "../simpleGrid/simpleGrid";
+import { Card as DCCard } from "../card/card";
+
+interface CardsGridProps {
+  records: CollectionDataType[] | ItemDataType[];
+}
+
+function isCollectionType(
```

Comment:
maybe make this an export here, or import it from lanes

Reply:
moved it to the utils file